denops/skkeleton/sources/deno_kv.ts

```diff
@@ -235,7 +238,7 @@ export class Dictionary implements BaseDictionary {
 
   private async loadMsgpack() {
     const data = await Deno.readFile(this.#path);
-    const jisyo = msgpack.decode(data) as Jisyo;
+    const jisyo = (msgpackDecode(data) as unknown) as Jisyo;
```

Comment:
Unnecessary brancket.
```suggestion
    const jisyo = msgpackDecode(data) as unknown as Jisyo;
```